gourmet/plugins/python_shell/__init__.py

```diff
@@ -1,4 +1,6 @@
 from gourmet.plugin import ToolPlugin
+from gi import require_version
+require_version("Gtk", "3.0")
```

Comment:
Any special reason this one has its own `require_version` call?

Reply:
I missed removing this `require_version`.